### ISSUE TITLE
[Programmatic Access - BOT Framework - Web Chat > Enter Keyword]: Elements having ARIA roles does not have required ARIA attributes.

### ISSUE THREAD

**Issue by vibhor12345678** [microsoft/BotFramework-WebChat#4341]
### User Experience:
Users who depend on screen readers will be impacted if aria attributes are not defined on the elements. Screen readers will not announce the information correctly and users will not get the proper information of the controls.

### Repro Steps:
1. Open URL: [Web Chat: Full-featured bundle (microsoft.github.io)](https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/)
2. Web chat bundle will open.
3. Navigate on the web chat bot and enter Help keyword in Type your message box, then select send button.
4. Cards related to the keyword will appear, navigate on the cards.
5. Verify the issue.

### Actual Result:
Elements having ARIA roles does not have required ARIA attributes.
Screen reader is announcing information as "Attachment Accessibility, Attachment Adaptive Card".

### Expected Result:
Ensures elements with ARIA roles have all required ARIA attributes.

### Issue: 
Ensures elements with ARIA roles have all required ARIA attributes (aria-required-attr - https://accessibilityinsights.io/info-examples/web/aria-required-attr)

### Target application: 
Web Chat: Full-featured bundle - https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/

### Element path: 
.webchat__stacked-layout__attachment-row[aria-roledescription="attachment"][role="group"]:nth-child(55) > .webchat__stacked-layout__attachment.webchat__bubble.webchat__bubble--nub-on-top > .webchat__bubble__content > .webchat--css-chppz-1kac3oh.webchat__adaptive-card-renderer > .ac-adaptiveCard.ac-container > .ac-container[dir="ltr"] > .ac-textBlock[role="heading"]

### How to fix: 
Fix any of the following:
  Required ARIA attribute not present: aria-level

### MAS Reference:
[MAS4.1.2- Name, Role, Value](https://aka.ms/MAS4.1.2)

### Reference Links
[Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Test Environment:
**Url:** [Web Chat: Full-featured bundle (microsoft.github.io)](https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/)
**Browser:** Chrome version 103.0.5060.114 (Official Build) (64-bit)
**OS:** Windows 11 version 21H2 (OS Build 22000.975)

**Comment by vibhor12345678** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS4.1.2;#Accessibility;#Win11-Chrome;#BotFrameworkWebChat-Jul22;

**Comment by compulim** [comment]
This is fixed in 4.15.3. It repro in 4.15.2.
